cmd/gcp-controller-manager/node_annotator.go

```diff
@@ -168,7 +171,7 @@ func (na *nodeAnnotator) enqueue(obj interface{}) {
 		utilruntime.HandleError(fmt.Errorf("couldn't get key for object %+v: %v", obj, err))
 		return
 	}
-	na.queue.Add(key)
+	na.queue.AddRateLimited(key)
```

Comment:
Leave this as Add. We don't want to backoff if there's a lot of node updates, and we are successfully processing them. We only want to backoff if there are errors